resources/sdk/purecloudjavascript/templates/configuration.mustache

```diff
@@ -0,0 +1,168 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+
+import Logger from './logger.js';
+
+class Configuration {
+	/**
+	 * Singleton getter
+	 */
+	get instance() {
+		return Configuration.instance;
+	}
+
+	/**
+	 * Singleton setter
+	 */
+	set instance(value) {
+		Configuration.instance = value;
+	}
+
+	constructor() {
+		if (!Configuration.instance) {
+			Configuration.instance = this;
+		}
+
+		this.configPath = typeof window !== 'undefined' ? '' : path.join(os.homedir(), '.genesyscloudjavascript', 'config');
+		this.refresh_access_token = true;
+		this.refresh_token_wait_max = 10;
+		this.live_reload_config = true;
+		this.host;
+		this.environment;
+		this.basePath;
+		this.authUrl;
+		this.config;
+		this.logger = new Logger();
+		this.setEnvironment();
+		this.liveLoadConfig();
+	}
+
+	liveLoadConfig() {
+		// If in browser, don't read config file, use default values
+		if (typeof window !== 'undefined') {
+			this.configPath = '';
+			return;
+		}
+
+		this.updateConfigFromFile();
+
+		if (this.live_reload_config && this.live_reload_config === true) {
+			try {
+				fs.watchFile(this.configPath, { persistent: false }, (eventType, filename) => {
+					this.updateConfigFromFile();
+					if (!this.live_reload_config) {
+						fs.unwatchFile(this.configPath);
+					}
+				});
+			} catch (err) {
+				// do nothing
+			}
+		}
+	}
+
+	setConfigPath(path) {
+		if (path && path !== this.configPath) {
+			this.configPath = path;
+			this.liveLoadConfig();
+		}
+	}
+
+	updateConfigFromFile() {
+		const ConfigParser = require('configparser');
```

Comment:
This line should be the only change compared to the previous PR. This line used to be at the top of the file, moved it here for browser compatibility.

Reply:
Is that all that needed to be fixed :D